airbyte-integrations/connectors/destination-postgres-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -342,6 +342,13 @@
         "description": "Drop tables with CASCADE. WARNING! This will delete all data in all dependent objects (views, etc.). Use with caution. This option is intended for usecases which can easily rebuild the dependent objects.",
         "title": "Drop tables with CASCADE. (WARNING! Risk of unrecoverable data loss)",
         "order": 11
+      },
+      "unconstrained_number": {
+        "type": "boolean",
+        "default": false,
+        "description": "Create numeric columns as unconstrained DECIMAL instead of NUMBER(38, 9). This will allow increased precision in numeric values.",
```

Comment:
nit: should we add something here about how this defaults to false for backwards compatibility, and we recommend setting this to true?

Reply:
added `(this is disabled by default for backwards compatibility, but is recommended to enable)` to the description